### DISCUSSION TITLE
Welcome to aem Discussions!

### DISCUSSION THREAD

**Discussion by jlentink** [jlentink/aem#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mayank-c-aggarwal** [comment]
Hi Jason,

I am using aem cli for my BAU tasks, in my password i have / character due to which aem cli not able to read my password due to which i am not able to use aem cli, and i have a limitation that i cannot change my password, what could be the solution for this please. getting below error:

Could not load config file. (could not decode config file: Near line 425 (last key parsed 'instance.password'): invalid escape character 'z'; only the following escape characters are allowed: \b, \t, \n, \f, \r, \", \\, \uXXXX, and \UXXXXXXXX)
